# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ash/slakware-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Cleanup from Slackware.

--- ./Makefile.orig	Sun May 15 14:16:31 1994
+++ ./Makefile	Tue Aug  3 16:19:10 1999
@@ -14,8 +14,10 @@
 
 OBJS =	$(OBJ1) $(OBJ2)
 
-CFLAGS = -O2 -fomit-frame-pointer -m486 -DSHELL -I/usr/include/bsd -I.
-LDFLAGS = -s -lbsd
+#CFLAGS = -O2 -pipe -DSHELL -I/usr/include/bsd -I. -D__BIT_TYPES_DEFINED__
+#LDFLAGS = -s -lbsd
+CFLAGS = -O2 -pipe -DSHELL -I. -D__BIT_TYPES_DEFINED__
+LDFLAGS = -s
 
 CLEANFILES =\
 	builtins.c builtins.h init.c mkinit mknodes mksyntax \
@@ -32,7 +34,7 @@
 token.def: mktokens
 	sh ./mktokens
 
-builtins.h builtins.c: mkbuiltins builtins
+builtins.h builtins.c: mkbuiltins
 	sh ./mkbuiltins
 
 init.c: mkinit $(SRCS)
--- ./mknodes.c.orig	Mon Sep  6 04:04:23 1993
+++ ./mknodes.c	Tue Aug  3 16:19:10 1999
@@ -89,7 +89,7 @@
 struct str *curstr;		/* current structure */
 
 
-FILE *infp = stdin;
+FILE *infp;
 char line[1024];
 int linno;
 char *linep;
@@ -102,6 +102,7 @@
 main(argc, argv)
 	char **argv;
 {
+	infp = stdin;
 	if (argc != 3) {
 		error("usage: mknodes file\n");
 		return(1);
--- ./trap.c.orig	Tue Aug 24 04:12:57 1993
+++ ./trap.c	Tue Aug  3 16:19:10 1999
@@ -53,6 +53,9 @@
 #include "mystring.h"
 #include <signal.h>
 
+#ifndef sig_t
+#define sig_t __sighandler_t
+#endif
 
 /*
  * Sigmode records the current value of the signal handlers for the various
